liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -163,10 +173,15 @@ void SourceReferenceFormatter::printSourceLocation(SourceReference const& _ref)
 
 void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg)
 {
-	// exception header line
-	errorColored() << _msg.severity;
-	if (m_withErrorIds && _msg.errorId.has_value())
-		errorColored() << " (" << _msg.errorId.value().error << ")";
+    if (_msg.severity.eq("Error") && _msg.errorId.has_value())
+        errorColored() << " (" << _msg.errorId.value().error << ")";
+
+    else if(_msg.severity.eq("Warning") && _msg.errorId.has_value())
+        warningColored() << " (" << _msg.errorId.value().error << ")";
+
+    else if(_msg.severity.eq("Info") && _msg.errorId.has_value())
```

Comment:
There's no `else`. What about messages where the severity is not one of these three values?

Reply:
I've added unknowsColored() function